cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -184,6 +185,13 @@ mkConsensusProtocolCardano NodeByronProtocolConfiguration {
            -- Alternatively, for testing we can transition at a specific epoch.
            --
            Just epochNo -> Consensus.TriggerHardForkAtEpoch epochNo)
+  where
+    -- The initial nonce, typically derived from the hash of Genesis config
+    -- JSON file.
+    --
+    -- WARNING: chains using different values of this parameter will be
+    -- mutually incompatible.
+    initialNonce = NeutralNonce
```

Comment:
This should be replaced with the hash of the Shelley genesis config JSON file, which should be taken as an argument/configuration parameter.

Reply:
Can we add a TODO?
```suggestion
    -- TODO: This should be replaced with the hash of the Shelley genesis config
    -- JSON file, which should be taken as an argument/configuration parameter.
    initialNonce = NeutralNonce
```